### ISSUE TITLE
Transforms Pipeline

### ISSUE THREAD

**Issue by arbelhizmi** [mahmoodlab/UNI#69]
Hi,

Regarding this pipeline:
`transform = transforms.Compose(
    [
        transforms.Resize(224),
        transforms.ToTensor(),
        transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
    ]
)` - is it assuming that images are PIL in `uint8` format? did you mean that images suppose to be in range [0, 1] before applying the `transform.Normalize` with the mean and std?

I'm using Xenium datasets and i want to extract features from pretrained UNIv2 model, but my images are already in size 224, already `torch.float32` and already tensors (automatically converted to tensors by `DataLoader`), so i'm wondering what you originally intended that the pipeline should be? because from what i understand UNIv2 is partially trained on Xenium datasets as well, and Xenium datasets usually are not cropped as `uint8` or PIL, unless you specifically crop them like this, so i'm not really sure what transformations should i use here..?

thanks
